recipes/gawk/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conan import ConanFile
+from conan.tools.files import copy, get, collect_libs
+from conan.tools.layout import basic_layout
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+import os
+
+required_conan_version = ">=1.52.0"
+
+
+class GawkConan(ConanFile):
+    name = "gawk"
+    description = "A program that you can use to select particular records in a file and perform operations upon them. GNU implementation of awk"
+    topics = ("file", "modification", "reformatting")
+    homepage = "http://git.savannah.gnu.org/cgit/gawk.git"
+    license = "GPL-3.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def configure(self):
+        # This is a C library. Remove C++ metadata
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        del self.info.settings.compiler
+
+    def compatibility(self):
+        return [{"settings": [("build_type", "Release")]}]
```

Comment:
```suggestion
    def compatibility(self):
        return [{"settings": [("build_type", "Release")]}]
```
Why do you need to customize the compatibility? Please, remove it as may result in missing packages in CCI in case consuming this one.